### DISCUSSION TITLE
Allow the calendar to automatically scroll so if it's defined in a card and the day has 20 entries the calendar smoothly scrolls through all entries.

### DISCUSSION THREAD

**Discussion by swampylee** [totaldebug/atomic-calendar-revive#1452] [Feature Requests]
As an example, the dakboard environment allows calendars to be defined and then scrolling set to slow / fast / none. This really helps with dashboard displays where you want to be able to see all upcoming events without the need to take up a huge amount of screen space.

**Comment by marksie1988** [comment]
this can be done with simple css modifications using card_mod, not something that i plan to implement.
